lib/app.js

```diff
@@ -13,80 +13,42 @@ import {
   ResponsiveWebapp,
   AppMenu
 } from 'otp-react-redux'
+
+// Loads a yaml config file which comes from a custom environment setting passed
+// into webpack or defaults to ./config.yml
 const otpConfig = require(OTP_CONFIG)
-export default class TrimetWebapp extends Component {
-  /**
-   * Get branding ID from config file. Defaults to TriMet.
-   */
-  _getBranding = () => otpConfig.branding || 'trimet'
 
-  _getCustomIcons = (branding) => {
-    let icons
-    switch (branding) {
-      case 'trimet':
-        icons = require('./icons/trimet')
-        break
-      default:
-        icons = require('./icons')
-        break
-    }
+// Loads a JavaScript file to perform extra configuration setup
+// FIXME figure out and document why it works to use a string template to load
+// the config.js file
+const jsConfig = require(`${otpConfig.jsConfigFile}`).configure(otpConfig)
```

Comment:
@evansiroky, thoughts on how to include a js config file that doesn't live in this repo? Or were you thinking we could just keep the various js config files in `lib/`?

Reply:
I just spent some time refactoring this and making another PR in our private repo that makes all this possible.